spec/models/mixins/authentication_mixin_spec.rb

```diff
@@ -692,6 +692,40 @@ def self.name; "TestClass"; end
           expect(@host.auth_user_pwd(:default)).to eq([@orig_ems_user, ''])
         end
       end
+
+      context "#change_password" do
+        let(:current_password) { "current_pass" }
+        let(:new_password) { "new_pass" }
+        let(:confirm_password) { "new_pass" }
+
+        it "should fail if some param is blank" do
+          current_password = ""
+          allow(@ems).to receive(:supports?).with(:change_password) { true }
```

Comment:
I'm wondering if it would be better, rather than stubbing the ems here, to build an ems that does support `change_password`? It would also be nice to see what happens to an ems that doesn't support `change_password`

Reply:
> It would also be nice to see what happens to an ems that doesn't support change_password

I think then just a test case for the unhappy path of trying to do this on an ems that doesn't support it would be great to add!

Reply:
[This line](https://github.com/ManageIQ/manageiq/pull/17323/files#diff-c2ad80b8d2756d1ea2122363614349c3R717) already does this.